NEWS.md

```diff
@@ -1,15 +1,16 @@
-# SKELETON
+# hySpc.read.spe 0.0.x 2021-xx-xx
 
 ## Acknowledgements
-The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during summer 2020.
-Erick and the team carried out a significant overhaul of `hyperSpec` which led to this release.
+The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during Summer 2020 and Sang Truong during summer 2021.
+Sang, Erick, and the team carried out a significant overhaul of `hyperSpec` which led to this release.
 
-# Version
+## User-Visible Changes
 
-## User-visible Changes
-
-### Bug Fixes
-### Breaking Changes
-### New Functions
-
-## Internal Changes
+* Moved functions from **hyperSpec** to **hySpc.read.spe**:
```

Comment:
1. Were the functions properly deprecated in `hyperSpec` package?

2. Was the statement that some functions were deprecated and their analogs can be found in `hySpc.read.spe` package included on `NEWS.md` of `hyperSpec` package?